lib/client.js

```diff
@@ -115,13 +115,45 @@ Client.prototype._request = function (method, ip, args, resultName) {
     });
 };
 
+Client.prototype._extractTransaction = function (result) {
+  const transaction = result['TRANS'];
+
+  if (transaction === null || transaction === undefined) {
+    return {};
+  }
+
+  const value = transaction['HPAY'];
+
+  if (value === null || value === undefined) {
+    return {};
+  }
+
+  return value;
+}
+
+Client.prototype._extractTransactionList = function (results) {
+  const transactions = results['TRANS'];
+
+  if (transactions === null || transactions === undefined) {
+    return [];
+  }
+
+  const value = transactions['HPAY'];
+
+  if (value === null || value === undefined) {
+    return [];
+  }
+
+  return value;
+}
```

Comment:
Is it really mandatory to use "Client" methods? 🤔 
```suggestion
function _extractTransaction(result) {
  const transaction = result['TRANS'];

  if (transaction === null || transaction === undefined) {
    return {};
  }

  const value = transaction['HPAY'];

  if (value === null || value === undefined) {
    return {};
  }

  return value;
}

function _extractTransactionList(results) {
  const transactions = results['TRANS'];

  if (transactions === null || transactions === undefined) {
    return [];
  }

  const value = transactions['HPAY'];

  if (value === null || value === undefined) {
    return [];
  }

  return value;
}
```

Reply:
I think you can do the changes ^^.

Reply:
Yeah I will do them, but I've others review to do before 🙈